### PR TITLE
fix: Add en-locale for those who are not set to en-US

### PR DESCRIPTION
**Pull request by ericawright**
Fixes: https://github.com/bwinton/TabCenter/issues/992.

### CODE REVIEW

File: genlocales [hunk 1]
```diff
@@ -29,6 +29,9 @@ glob('locales/*/*.properties', function (err, files) {
     if (match && lang !== 'en-US') {
       descriptions[lang] = {'description': match[1]};
     }
+    if (lang === 'en-US') {
+      fs.createReadStream(path).pipe(fs.createWriteStream(`locale/en.properties`));
+    }
```
Comment (bwinton) [inline genlocales]: What if we just renamed the `en-US` directory `en` instead?

**Comment by bwinton** [comment]
(Also, @flodolo might know whether the "rename `en-US` to `en`" solution would have any hidden problems…)

**Comment by flodolo** [comment]
https://developer.mozilla.org/en-US/docs/Mozilla/Implementing_Pontoon_in_a_Mozilla_website

> Source locale needs to be called templates, en or en-US.

I think Pontoon will just read the strings from en, and nothing bad should happen in Pontoon. CC @mathjazz just in case.

Personally I think the approach suggested in this patch makes more sense than having a `en` folder. To give you an idea, the [only project](https://l10n.mozilla-community.org/webstatus/?locale=en) using 'en' is Firefox Account.

**Comment by mathjazz** [comment]
> I think Pontoon will just read the strings from en, and nothing bad should happen in Pontoon. CC @mathjazz just in case.

Confirmed.

> Personally I think the approach suggested in this patch makes more sense than having a en folder. To give you an idea, the only project using 'en' is Firefox Account.

I also agree with that.

**Comment by bwinton** [comment]
Okay, I'll merge this as soon as @ericawright updates it to the latest `master`.  Thanks!

**Pull request merged**